tests/ui/llm_studio.feature

```diff
@@ -0,0 +1,19 @@
+Feature: LLM Studio
+
+    Background: LLM Studio user
+        Given LLM Studio home page is opened
+        When I login to LLM Studio
+        Then I see the home page
+
+    Scenario: Import dataset using filesystem
+        When I upload dataset with path /home/llmstudio/mount/data/user/oasst/train_full.pq and name test_dataset
```

Comment:
looks like deployment-specific file path. could it be something like:
```gherkin
When I upload dataset oasst/train_full.pq as test_dataset
```
And default location could different for local and remove test execution?